libs9s/s9srpcreply.cpp

```diff
@@ -347,6 +347,46 @@ S9sRpcReply::users()
     return theList;
 }
 
+S9sVariantList
+S9sRpcReply::dbVersions()
+{
+    S9sVariantList  theList;
+    
+    if (contains("db_versions"))
+        theList = operator[]("db_versions").toVariantList();
+    else if (contains("db_versions"))
```

Comment:
same `contains("db_versions")` condition in `if` and its `esle if`, why it is so?